website/src/documentation/sdks/java-dependencies.md

```diff
@@ -26,336 +26,24 @@ behavior in the service. If you are using any of these packages in your code, be
-  <tr><td>joda-time</td><td>joda-time</td><td>2.4</td></tr>
-  <tr><td>junit</td><td>junit</td><td>4.12</td></tr>
-  <tr><td>org.apache.kafka</td><td>kafka_2.11</td><td>1.0.0</td></tr>
-  <tr><td>org.apache.kafka</td><td>kafka-clients</td><td>1.0.0</td></tr>
-  <tr><td>org.apache.apex</td><td>malhar-library</td><td>3.4.0</td></tr>
-  <tr><td>org.mockito</td><td>mockito-core</td><td>1.10.19</td></tr>
-  <tr><td>io.netty</td><td>netty-handler</td><td>4.1.25.Final</td></tr>
-  <tr><td>io.netty</td><td>netty-tcnative-boringssl-static</td><td>2.0.8.Final</td></tr>
-  <tr><td>io.netty</td><td>netty-transport-native-epoll</td><td>4.1.25.Final</td></tr>
-  <tr><td>org.postgresql</td><td>postgresql</td><td>42.2.2</td></tr>
-  <tr><td>org.powermock</td><td>powermock-mockito-release-full</td><td>1.6.4</td></tr>
-  <tr><td>com.google.protobuf</td><td>protobuf-java</td><td>3.6.0</td></tr>
-  <tr><td>com.google.protobuf</td><td>protobuf-java-util</td><td>3.6.0</td></tr>
-  <tr><td>com.google.api.grpc</td><td>proto-google-cloud-pubsub-v1</td><td>1.18.0</td></tr>
-  <tr><td>com.google.api.grpc</td><td>proto-google-cloud-spanner-admin-database-v1</td><td>0.19.0</td></tr>
-  <tr><td>com.google.api.grpc</td><td>proto-google-common-protos</td><td>1.12.0</td></tr>
-  <tr><td>org.slf4j</td><td>slf4j-api</td><td>1.7.25</td></tr>
-  <tr><td>org.slf4j</td><td>slf4j-jdk14</td><td>1.7.25</td></tr>
-  <tr><td>org.slf4j</td><td>slf4j-log4j12</td><td>1.7.25</td></tr>
-  <tr><td>org.slf4j</td><td>slf4j-simple</td><td>1.7.25</td></tr>
-  <tr><td>org.xerial.snappy</td><td>snappy-java</td><td>1.1.4</td></tr>
-  <tr><td>org.apache.spark</td><td>spark-core_2.11</td><td>2.3.2</td></tr>
-  <tr><td>org.apache.spark</td><td>spark-network-common_2.11</td><td>2.3.2</td></tr>
-  <tr><td>org.apache.spark</td><td>spark-streaming_2.11</td><td>2.3.2</td></tr>
-  <tr><td>org.codehaus.woodstox</td><td>stax2-api</td><td>3.1.4</td></tr>
-  <tr><td>org.codehaus.woodstox</td><td>woodstox-core-asl</td><td>4.4.1</td></tr>
-  <tr><td>com.pholser</td><td>junit-quickcheck-core</td><td>0.8</td></tr>
-</table>
+    Replace `<version_number>` with the major.minor.patch version of the SDK. For example, <https://raw.githubusercontent.com/apache/beam/v2.18.0/buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy>{:target="_blank"} will provide the dependencies for the 2.18.0 release.
+    
+2. Review the list under `project.ext.library`. 
 
-</details>
+    **Note:** Some dependencies in the list use version variables, such as `google_auth_version`. These variables are defined prior to the `project.ext.library` map definition.
 
-<details><summary markdown="span"><b>2.8.0</b></summary>
+You can also retrieve this list through Maven by running the following from the root of your project directory:
 
-<p>Beam SDK for Java 2.8.0 has the following compile and runtime dependencies.</p>
+```java
+$ mvn dependency:resolve && mvn -o dependency:list
```

Comment:
Note that this will get the dependencies of the *current* project. If they already have a dependency conflict it will simply show the version that is installed in that project. We would need to compare the versions with a clean project to see any differences.

If we want to get them from a new project we would need to create one:
```
export BEAM_VERSION=2.18.0
export JAVA_VERSION=11

mvn archetype:generate \
    -DinteractiveMode=false \
    -DarchetypeGroupId=org.apache.beam \
    -DarchetypeArtifactId=beam-sdks-java-maven-archetypes-starter \
    -DarchetypeVersion=$BEAM_VERSION \
    -DtargetPlatform=$JAVA_VERSION \
    -DartifactId=check-pipeline-dependencies \
    -DgroupId=org.apache.beam.samples

cd check-pipeline-dependencies

mvn dependency:resolve && mvn -o dependency:list
```